keps/sig-apps/2255-pod-cost/README.md

```diff
@@ -55,101 +48,133 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
-* New E2E Tests to validate that replicas get deleted as expected e.g:
- * Replicas with lower pod-cost before replicas with higher pod-cost
- * Replicas with no pod-cost annotation set before replicas with low priority
+* Integration tests to validate that:
+ * Replicas with lower pod-cost are deleted before replicas with higher pod-cost
+ * No behavior change when pod-cost is not set or all pods have the same pod-cost
 
 ### Graduation Criteria
 
 #### Alpha -> Beta Graduation
 * Implemented feedback from alpha testers
-* Thorough E2E and unit testing in place
 
 #### Beta -> GA Graduation
-* Significant number of end-users are using the feature
 * We're confident that no further API changes will be needed to achieve the goals of the KEP
 * All known functional bugs have been fixed
 
 ### Upgrade / Downgrade Strategy
 
-When upgrading no changes are needed to maintain existing behaviour as all of this behaviour is fully optional and disabled by default. To activate this feature either a user has to make an annotation to a pod in a Deployment by hand or the application annotates a pod in a Deployment through the API.
-
-When downgrading, there is no need to changing anything, as this is just a pod annotation, which is uncritical.
+N/A
 
 ### Version Skew Strategy
 
-As this feature is based on pod annotations, there is no issue with different Kubernetes versions. The lack of this feature in older versions may change the efficiency and reliability of the applications.
+N/A
 
 ## Production Readiness Review Questionnaire
 
 ### Feature enablement and rollback
 
 * **How can this feature be enabled / disabled in a live cluster?**
-  - [x] Other
-    - Make special pod annotations within a live Deployment
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: ReplicaSetPodDeletionCost
+    - Components depending on the feature gate: kube-controller-manager
+  - [ ] Other
+    - Describe the mechanism:
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
 
 
 * **Does enabling the feature change any default behavior?**
-  - No
+ No.
 
 
 * **Can the feature be disabled once it has been enabled (i.e. can we rollback
   the enablement)?**
-  - One can either remove the annotations or downgrade to an older Kubernetes release
+Yes.
 
 
 * **What happens if we reenable the feature if it was previously rolled back?**
-  - Then the feature will be reenabled. Nothing special to consider here.
+It should continue to work as expected.
 
 
 * **Are there any tests for feature enablement/disablement?**
+No, but we can do manual testing.
 
+## Implementation History
+- 2021-01-13: Initial KEP submitted as provisional
+- 2021-01-15: KEP promoted to implementable
+
+
+<!--
```

Comment:
There are still some mentions of "priority" in this doc. Please update those to pod cost.

Reply:
done.